2018-05-23-Meeting.md

```diff
@@ -1,25 +1,54 @@
+# Present
+Karin Lagesen
+Øivind Øines
+Arne Holst Jensen
+Girum Tadesse Tessema
+Hege Divon
+Thomas Haverkamp
+Maria Dahle
+Håkon Kaspersen
+
 # Agenda
 
 ## Communications
 
 In case of questions, send email to the abelgroup email list.
+- Any questions are allowed, none are stupid!
+- This may lead to quicker responses, and also informs the rest of the group
 
 ## Information files
 Several directories have read-me files, read them!
+- If there is a readme file in a directory, read it!
+- The readme file describes catalogue structures, what is supposed to be inside the catalogue in question, etc.
 
 Also: new feature: NEWS file
+- The news file contain information on new programs, how to find them, etc.
+- Also contains information on programs that doesn't work a the moment
+- The person that updates the news file should also send an email to the abel email list
 
 ## Folder stucture
 
 1. Directories
   * "Landing directory", like "/usit/abel/u1/karinlag" are free for your use, but no VI data should be kept there
   * All should have their own project home directory, this would be "/work/projects/nn9305k/home/karinlag".
+    - All Veterinary Institute data should only be in the nn9305k project catalogue. We have a total of 20 TB of space available in that 
+catalogue.
```

Comment:
20 TB in total on abel, in two different directories, this one and a backup directory.